ext/java/org/msgpack/jruby/Encoder.java

```diff
@@ -45,6 +46,14 @@ public Encoder(Ruby runtime, boolean compatibilityMode) {
     this.compatibilityMode = compatibilityMode;
   }
 
+  public void setRegistry(Packer.ExtensionRegistry registry) {
+    this.registry = registry;
+  }
```

Comment:
It can't be, because registries will be overwritten when initialized from factories.
https://github.com/msgpack/msgpack-ruby/blob/ext-type-jruby/ext/java/org/msgpack/jruby/Packer.java#L114